backend/frontend/templates/frontend/user/board/partial_card.html

```diff
@@ -1,170 +1,225 @@
-        <div class="flex gap-2 items-center">
-            <i class="fa-regular fa-brands fa-github fa-lg"></i>
-            <a class="{{styles.link}}" target="_blank" href="{{card.recruit_project.repository.get_github_repo_link}}">Repository</a>  
-        </div>
-    {% endif %}
+    <!-- Assignee and reviewers section -->
 
-    {% if card.request_user_can_start %}
-        <button type="button" class="{{ styles.button_primary_small }}"
+    <div class="grid grid-cols-2 grid-rows-1 gap-x-10 gap-y-3 w-full h-[auto] pt-2">
+
+      <div class="Assignee-avatar col-span-1 row-span-1 group inline-block">
+        <div class="mb-2 text-xs text-indigo-950 font-['Poppins']">Assignee</div>
+        {% if card.assignee_names %}
+          {% for assignee in card.assignee_names %}
+            <img
+              class="w-[30px] h-[30px] rounded-full"
+              src="https://images.unsplash.com/photo-1594751543129-6701ad444259?q=80&w=1000&auto=format&fit=crop&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxzZWFyY2h8Mnx8ZGFyayUyMHByb2ZpbGV8ZW58MHx8MHx8fDA%3D"
+              alt="{{ assignee }} profile picture"
+              title="{{ assignee }}"
+            />
+          {% endfor %}
+        {% endif %}
+      </div>
+
+      <div class="Reviewer-avatars col-span-1 row-span-1 group">
+        <div class="mb-2 text-xs text-indigo-950 font-['Poppins']">Reviewers</div>
+          <div class="flex relative">
+            {% if card.reviewer_names %}
+                {% for reviewer in card.reviewer_names|slice:":2" %}
+                    <div class="flex -mx-2 overflow-hidden rounded-full border-2 border-white">
+                        <img class="w-[30px] h-[30px] rounded-full" 
+                            src="https://img.freepik.com/premium-photo/portrait-smiling-happy-multiethnic-elementary-school-boy-dressed-formal-school-uniform-wit_616370-5177.jpg" 
+                            alt="{{ reviewer }} profile picture" 
+                            title="{{ reviewer }}"
+                        />
+                    </div>
+                {% endfor %}
+                {% if card.reviewer_names|length > 2 %}
+                  <div class="group">
+                    <div class="flex -mx-2 overflow-hidden rounded-full border-2 border-white cursor-pointer">
+                      <img class="w-[30px] h-[30px] rounded-full" 
+                        src="https://cdn.pixabay.com/photo/2015/10/05/22/37/blank-profile-picture-973460_960_720.png" 
+                        alt="more reviewers" 
+                        title="{{ card.reviewer_names|join:', ' }}"
+                      />
+                    </div>
+                  </div>
+                {% endif %}
+            {% endif %}
+        </div>
+      </div>      
+    
+      <!-- Buttons section -->
+      
+      <div class="grid grid-cols-2 grid-auto-rows-1fr gap-x-10 gap-y-10 w-full h-[auto]">
+        <div class="Start-button col-span-1 row-span-1 w-full">
+          <button
```

Comment:
Clicking on the content button should not start the card. It should rather link to the card content and open it in a new tab.   

Alternatively,  `alert("TODO")` so it's very obvious that it needs to be fixed, and you can also make a Mom task so it can be done

Please don't make buttons do random things!